systems/plants/TimeSteppingRigidBodyManipulator.m

```diff
@@ -620,7 +620,8 @@ function checkDirty(obj)
 
```

Comment:
the fastqp logic above looks all wrong to me.  
first off, Aeq should be using M_active, not M_inactive, right?  

but i would actually think that you'd want to tell fastqp that they are all inequality constraints, and pass in the *likely* equality constraints as the active set.  e.g., something like the code below (typed in off the top of my head):

```
Aeq = [];
beq = [];
Ain_fqp = -[M;eye(nz)];
bin_fqp = -[w;lb];
QblkDiag = {eye(nz)};
fqp = zeros(nz, 1);
fastqp_active_set = [z_inactive;~z_inactive]; 
[z,info_fqp] = fastQPmex(QblkDiag,fqp,Ain_fqp,bin_fqp,Aeq,beq,fastqp_active_set);
```